frontend/tests/frontend/helloworld.js

```diff
@@ -0,0 +1,9 @@
+import { helloWorld } from "../../src/js/helloworld";
```

Comment:
We can only use ES6 imports in this Jasmine script because the `yarn test` command contains a compilation command that uses Parcel to compile this test file (see above). Otherwise, Jasmine won't understand ES6 syntax.